std/format.d

```diff
@@ -1995,6 +1995,7 @@ private void formatUnsigned(Writer, T, Char)
             + zerofill
             + digits.length
             + ((fs.flSeparator != 0) * (digits.length / fs.separators))
+            - (fs.flSeparator != 0 && digits.length % fs.separators == 0)
```

Comment:
`((fs.flSeparator != 0) * ((digits.length - 1) / fs.separators))` replaces the two lines and is quite a bit simpler

Reply:
fxied